common/lib/core-interfaces/src/fluidObject.ts

```diff
@@ -3,36 +3,10 @@
  * Licensed under the MIT License.
  */
 
-import {
-    IFluidLoadable,
-    IFluidRunnable,
-} from "./fluidLoadable";
-import { IFluidRouter } from "./fluidRouter";
-import { IFluidHandle, IFluidHandleContext } from "./handles";
-
 /**
  * @deprecated Use `FluidObject` or the interface directly instead
  */
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
 export interface IFluidObject {
```

Comment:
When does this interface go away entirely?  Maybe update the deprecated comment above to indicate?

Reply:
i call it out in the breaking.md. some people still augment it, so it needs to stick around, but we don't us it anywhere